src/modules/Device/device.types.ts

```diff
@@ -34,8 +34,9 @@ interface DeviceConstructorParams {
   // options?: any;
 }
 
-type DataToSend = Omit<Data, "id" | "created_at" | "origin" | "device" | "time"> & { time?: Date | string };
-type DataToEdit = Omit<Data, "created_at" | "origin" | "device" | "time"> & { time?: Date | string };
+type DataToSend = DataCreate;
+type DataToEdit = DataEdit;
```

Comment:
Using the types from `commons` for these `DataToSend` and `DataToEdit` types just for compatibility if anyone imported them.

This can be removed.